src/features/courses/course-controller.ts

```diff
@@ -3362,6 +3363,63 @@ class CourseController {
             return result;
         });
     }
+
+    // Sends an email to the professor associated with the course.
+    async emailProfessor(options: EmailProfOptions): Promise<boolean> {
+        const coursePromise = Course.findOne({
+            where: {
+                id: options.courseId,
+            },
+            include: [{
+                model: User,
+                as: 'instructor',
+                attributes: ['id', 'firstName', 'lastName', 'email', 'preferredEmail'],
+                where: {
+                    active: true
+                },
+                required: true,
+            }],
+        });
+
+        const gradePromise = StudentGrade.findOne({
+            where: {
+                userId: options.student.id,
+                courseWWTopicQuestionId: options.question.id,
+            },
+            attributes: ['randomSeed']
+        });
+
+        const [course, grade] = await Promise.all([coursePromise, gradePromise]);
+
+        if (_.isNil(course) || _.isNil(course.instructor)) {
+            throw new WrappedError('Could not find an instructor for this course.');
+        }
+
+        if (_.isNil(grade)) {
+            throw new WrappedError('Could not find a grade associated with the problem this student is trying to ask about.');
+        }
+
+        const poorMansTemplate = `
+Hello Professor ${course.instructor.lastName},
+
+    Your student ${options.student.firstName} ${options.student.lastName} is asking for help with
+Problem ${options.question.problemNumber+1} (ID#${options.question.id}) in the Topic ${options.topic.name} (ID#${options.topic.id})
```

Comment:
Why are we incrementing problemNumber by 1?

Also I still don't like the id int the request, it's great for reordering but i'm skeptical on putting these numbers in there

Reply:
This is user-facing, so it needs to be offset by one.

It doesn't hurt anything to have more information right now. If the professor can see the ID numbers anywhere, then they can use this. I would only remove it if the information was misleading, like that the ID messages don't match what's on the Settings page.

Reply:
problem number should be misleading if it's not there is probably a bigger issue here

Reply:
As @doombot-exe pointed out yesterday too much information can be overbearing, the only advantage to showing the id here is if the problem has been reordered